backend/controllers/appointments/confirmAppointment.js

```diff
@@ -22,7 +22,10 @@ const confirmAppointmentFactory = ({ prisonApi, appointmentsService, logError })
       )
 
       const appointmentDetails = req.flash('appointmentDetails')
-      if (!appointmentDetails?.length) throw new Error('Appointment details are missing')
+      if (!appointmentDetails?.length) {
+        res.redirect(prisonUser ? `/prisoner/${offenderNo}` : '/videolink/prisoner-search')
```

Comment:
Be better to do a return res.redirect

Reply:
Ha - that's what i did originally but eslint complained that it was an inconsistent return type (there is no return value otherwise) - https://app.circleci.com/pipelines/github/ministryofjustice/prisonstaffhub/3861/workflows/4382c341-35af-4ca4-ab78-4da536879af5/jobs/17620.